### PR TITLE
Added Travis CI

### PR DESCRIPTION
**Pull request by Aniq55**

### CODE REVIEW

**Comment by benkiel** [comment]
@Aniq55 This looks like it's failing because the tests aren't also installing fontparts — I took a quick pass at it to see if I could get it to import, but it's still not quite working, could you take a look and see if you can find what I missed? Thank you!

**Comment by Aniq55** [comment]
@benkiel Sure.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: python
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+  - python setup.py build
```
Comment (anthrotype) [inline .travis.yml:5]: `setup.py build` is not needed. It's always implied by `setup.py install`

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  - python setup.py install
```
Comment (anthrotype) [inline .travis.yml:6]: `setup.py install` is deprecated (uses setuptools's `easy_install` command, produces an `.egg` file), you should use pip to install, which will internally create a wheel and install from there.

You can install from a local directory (e.g. the current one where setup.py is located) with `pip install .`

**Comment by anthrotype** [comment]
@Aniq55 I suggest you have a read at https://packaging.python.org/

also try to see how other related projects are doing this.

File: .travis.yml (hunk 1 above)
Comment (Aniq55) [reply .travis.yml:5]: alright.

**Comment by Aniq55** [comment]
@anthrotype Sure, will go through it and attempt this CI again.

**Comment by benkiel** [comment]
Done in #30

**Pull request closed without merging**